applicationinsights.json

```diff
@@ -6,13 +6,19 @@
     "service.version": "${BUILD_NUMBER}"
   },
   "instrumentation": {
+    "springScheduling": {
+      "enabled": false
+    },
     "logging": {
       "level": "DEBUG"
     }
   },
   "selfDiagnostics": {
     "destination": "console"
   },
+  "sampling": {
```

Comment:
this will grab all requests, not just a sample of them.

Reply:
Could this be the problem?

Reply:
based it on [the](https://github.com/ministryofjustice/hmpps-restricted-patients-api/blob/main/applicationinsights.json). Should I remove the sampling bit altogether?